R/transform-files.R

```diff
@@ -256,14 +256,21 @@ parse_transform_serialize_r <- function(text,
     transformers
   )
 
+  strict <- transformers$more_specs_style_guide$strict
   pd_split <- unname(split(pd_nested, pd_nested$block))
   pd_blank <- find_blank_lines_to_next_block(pd_nested)
 
+  start_line <- if (strict) {
+    1L # FIXME: should be 0L for roxygen examples
```

Comment:
@lorenzwalthert Having hard time figuring out if code is roxygen example from here. Any tips?

Reply:
Also, do you want to always convert
````r
#' @examples 1
```
to 
```r
#' @examples
#' 1 
```
Or not? I tried to avoid this so far, but if it's impossible to keep as is, we could also discuss moving always to the next line

Reply:
I'd prefer if we stick to the current behaviour. That is, we always style to the following, even if there is just one line of code in examples section:

```r
#' @examples
#' 1 
```

The changes here are unintentional.

Reply:
> in `style_roxygen_example_snippet()`, we call `parse_transform_serialize_r()`. Hence, we could add a new argument to the latter `is_roxygen_code_example` and set it to `TRUE` when calling `parse_transform_serialize_r()` from `style_roxygen_example_snippet()`.

Thanks for the tip! Will proceed accordingly.